### PR TITLE
Minor text fixes

### PR DESCRIPTION
**Pull request by eread-usgs**

### CODE REVIEW

File: viz.yaml [hunk 1]
```diff
@@ -309,7 +309,7 @@ publish:
       id: header
       title: Hurricane Harvey's Water Footprint
       logo: usgs-logo
-      caption: Water impacts from Hurricane Harvey
+      caption: Water impacts from Harvey Matthew
```
Comment (jordansread) [inline viz.yaml]: you need to `git pull upstream master`

File: viz.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-vizlab: "0.1.7"
+vizlab: "0.1.4"
```
Comment (jordansread) [inline viz.yaml]: this should be 0.1.7

File: viz.yaml (hunk 1 above)
Comment (eread-usgs) [reply viz.yaml]: now OK @jread-usgs ?

**Comment by eread-usgs** [comment]
@ldecicco-USGS can you take a look at this and see if i resolved the issue?

**Pull request merged**